dashboard/src/pages/TreeDetails/treeDetailsUtils.ts

```diff
@@ -0,0 +1,20 @@
+import type { TFilter, TFilterKeys } from '@/types/tree/TreeDetails';
+
+export const cleanFalseFilters = (diffFilter: TFilter): TFilter => {
+  const cleanedFilter: TFilter = JSON.parse(JSON.stringify(diffFilter ?? {}));
+  Object.entries<Record<string, boolean>>(cleanedFilter).forEach(
+    ([filterSectionKey, filterSectionValue]) => {
+      Object.entries(filterSectionValue).forEach(([filterKey, filterValue]) => {
+        if (filterValue === false) delete filterSectionValue[filterKey];
+      });
+
+      if (
+        Object.keys(filterSectionValue).length === 0 &&
+        cleanedFilter[filterSectionKey as TFilterKeys]
+      ) {
+        delete cleanedFilter[filterSectionKey as TFilterKeys];
+      }
+    },
+  );
+  return cleanedFilter;
```

Comment:
How about we iterate over `diffFilter` and construct the `cleanedFilter` in the process instead of creating the `cleanedFilter` and then deleting the properties?

Reply:
fixed in
https://github.com/kernelci/dashboard/pull/93/commits/d1fd009e2c70dd340f5e557bc17ea6cf2b971c1d